### PR TITLE
Add instructions for webpack 2

### PR DESCRIPTION
**Pull request by silvenon**
https://gist.github.com/sokra/27b24881210b56bbaff7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,21 @@ if (!global.Intl) {
 
 _note: the locale data is required for the polyfill to function when using it in a browser environment, in the example above, the english (`en`) locale is being required along with the polyfill itself._
 
+If you're using [webpack 2], instead of `require.ensure(...)` use `System.import`:
+
+```javascript
+// ...
+
+    Promise.all([
+        System.import('intl'),
```
Comment (caridy) [inline README.md]: AFAIK webpack 2 is experimental. Also, I'm the editor of the Loader Spec, and `System.import` does not exist in that spec yet, only `System.loader.import`, not sure where is that coming from.

Comment (silvenon) [inline README.md]: Not experimental, it's in beta, people are starting to use it. I don't have strong opinions about it, I just thought this info might be useful. I'm not sure about why `System.import` is not in the spec, but that's what webpack 2 uses… Feel free to close 😉

Comment (Delagen) [inline README.md]: @silvenon I tried to use it, but simply cannot get loaders to work as expected, even if they provided by webpack team

**Comment by silvenon** [comment]
Closing this as webpack v2 is still in beta.

**Pull request closed without merging**